dev-libs/libbpf/libbpf-9999.ebuild

```diff
@@ -10,44 +10,35 @@ if [[ ${PV} =~ [9]{4,} ]]; then
 	EGIT_REPO_URI="https://github.com/libbpf/libbpf.git"
 else
 	SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64 ~arm64 ~x86"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
 fi
 S="${WORKDIR}/${P}/src"
 
-HOMEPAGE="https://github.com/libbpf/libbpf"
 DESCRIPTION="Stand-alone build of libbpf from the Linux kernel"
+HOMEPAGE="https://github.com/libbpf/libbpf"
 
 LICENSE="GPL-2 LGPL-2.1 BSD-2"
 SLOT="0/${PV}"
-IUSE="+static-libs"
 
-COMMON_DEPEND="virtual/libelf
-	!<=dev-util/bcc-0.7.0"
-DEPEND="${COMMON_DEPEND}
-	sys-kernel/linux-headers"
-RDEPEND="${COMMON_DEPEND}"
+DEPEND="
+	sys-kernel/linux-headers
+	virtual/libelf"
+RDEPEND="${DEPEND}"
 
 PATCHES=(
-	"${FILESDIR}/libbpf-9999-paths.patch"
+	"${FILESDIR}"/libbpf-9999-paths.patch
 )
 
-src_compile() {
+src_configure() {
 	append-cflags -fPIC
-	emake \
-		BUILD_SHARED=y \
-		LIBSUBDIR="$(get_libdir)" \
-		$(usex static-libs 'BUILD_STATIC=y' '' '' '') \
```

Comment:
Did they remove this option or did BUILD_STATIC=y never do what it looks like?

Reply:
Yeah they removed it, now there is only `BUILD_STATIC_ONLY` so the static library gets built either way